crates/bevy_pbr/src/render/mesh_functions.wgsl

```diff
@@ -12,6 +12,7 @@
 }
 #import bevy_render::maths::{affine3_to_square, mat2x4_f32_to_mat3x3_unpack}
 
+#ifndef MESHLET_MESH_MATERIAL_PASS
```

Comment:
Why were the `#ifndef MESHLET_MESH_MATERIAL_PASS` added here? Did they cause compilation issues?

Reply:
The `mesh` array that's used in those functions isn't present during visbuffer resolution.

Reply:
Right, but I don't think that causes any actual issues, unless you try and use them? It was never an issue in 0.15. Unless a recent PR changed that. Anyways I'm fine with the ifdef, I was just wondering why it was added.

Reply:
It caused the shaders to fail to compile.